static/app/utils/replays/replayReader.tsx

```diff
@@ -259,21 +257,6 @@ export default class ReplayReader {
     ].sort(sortFrames)
   );
 
-  countDomNodes = memoize(() =>
-    countDomNodes({
-      frames: this.getRRWebMutations(),
-      rrwebEvents: this.getRRWebFrames(),
-      startTimestampMs: this._replayRecord.started_at.getTime(),
-    })
-  );
-
-  getDomNodes = memoize(() =>
-    extractDomNodes({
-      frames: this.getDOMFrames(),
-      rrwebEvents: this.getRRWebFrames(),
-    })
-  );
```

Comment:
These are both memoized here, but also memoized again when we leverage `useApiQuery` to await the promises.

So i thought, we can remove this layer of the callstack.